src/padding/_menu.scss

```diff
@@ -16,7 +16,7 @@
   }
 }
 
-menupopup:not(.in-menulist) > menuitem,
+menupopup > menuitem,
```

Comment:
`.in-menulist` is known to affect the OS, so I'll test it on VM tomorrow. #227
I don't remember exactly, but it requires testing on Windows 7, 10, and Linux.

Reply:
forgot to say I have only tested this on linux